config/settings/integration_tests.py

```diff
@@ -73,56 +66,9 @@
     "DJANGO_SECRET_KEY", default="RPr%].dHP4VLcU}SW(5o4;cH(]4i7?noV.q5*.%16!@#TYO/ku"
 )
 
-# AWS
-AWS_ACCESS_KEY_ID = env("AWS_ACCESS_KEY_ID")
-AWS_SECRET_ACCESS_KEY = env("AWS_SECRET_ACCESS_KEY")
-AWS_S3_BUCKET = env("AWS_S3_BUCKET")
-
-# Salesforce
-SALESFORCE_CLIENT_ID = env("SALESFORCE_CLIENT_ID")
-SALESFORCE_JWT_PRIVATE_KEY = process_key(env("SALESFORCE_JWT_PRIVATE_KEY"))
-SALESFORCE_SANDBOX = env.bool("SALESFORCE_SANDBOX")
-SALESFORCE_USERNAME = env("SALESFORCE_USERNAME")
-SALESFORCE_ARTICLE_AUTHOR_FIELD = env("SALESFORCE_ARTICLE_AUTHOR_FIELD")
-SALESFORCE_ARTICLE_AUTHOR_OVERRIDE_FIELD = env(
-    "SALESFORCE_ARTICLE_AUTHOR_OVERRIDE_FIELD"
-)
-SALESFORCE_ARTICLE_TYPE = env("SALESFORCE_ARTICLE_TYPE")
-SALESFORCE_ARTICLE_BODY_FIELD = env("SALESFORCE_ARTICLE_BODY_FIELD")
-SALESFORCE_ARTICLE_URL_PATH_PREFIX = env(
-    "SALESFORCE_ARTICLE_URL_PATH_PREFIX", default="/articles/Resource/"
-)
-SALESFORCE_ARTICLE_TEXT_INDEX_FIELD = env(
-    "SALESFORCE_ARTICLE_TEXT_INDEX_FIELD", default=False
-)
-SALESFORCE_ARTICLE_LINK_LIMIT = env("SALESFORCE_ARTICLE_LINK_LIMIT", default=100)
-SALESFORCE_ARTICLE_PREVIEW_URL_PATH_PREFIX = '/preview'
```

Comment:
Please confirm it's okay for this one to change

Reply:
It was safe for it to change but there was no reason for it to do so.

873d9bb